### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by nikolaswise**
@jgravois This fixes a number of problematic org methods, adds item methods to get calls within other item methiods, and gets us on track for more stable semvr releases

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-Copyright (c) 2015, Esri PDX R&D
```
Comment (jgravois) [inline LICENSE.md]: why remove the license file completely?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file. This projec
 
 ## [Upcoming changes][unreleased]
 
+## 1.0.1 - 04.18.2016
```
Comment (jgravois) [inline CHANGELOG.md]: lets add links so people can see commits between versions too..

```
[unreleased]: https://github.com/esripdx/node-arcgis/compare/v1.0.0...HEAD
[1.0.1]: https://github.com/esripdx/node-arcgis/compare/v1.0.0-beta.1...v1.0.1
[1.0.0-beta.1]: https://github.com/esripdx/node-arcgis/tags/v1.0.0-beta.1
```

Comment (nikolaswise) [inline CHANGELOG.md]: That is a cool idea which I like.

File: README.md [hunk 3]
```diff
@@ -9,17 +9,44 @@
...
+var ago = ArcGIS({token: 'usertoken'})
 ```
+
 ## Documentation
 
-> http://node-arcgis.surge.sh/ - API Reference and Examples
+API Reference and Examples
+https://node-arcgis.surge.sh
+
+## Contributing
```
Comment (jgravois) [inline README.md]: gotta get rid of the first two 'Contributing' and 'License' sections again...

**Pull request merged**